### PR TITLE
Update flutterflow doc

### PR DESCRIPTION
**Pull request by kawofong**
Updated FlutterFlow documentation in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ We are building core services used for GenAI and Enterprise Search use cases, wi
 flexibility to leverage a number of LLMs like VertexAI, ChatGPT 3.5/4, Cohere, and more to come.
 
 Highlighted features:
+
 - Enable any enterprise to leverage a chatbot to answer any questions based on their selected
   content such as a list of documents, databases, etc.
 - Easy to plug in and use any LLM models available in the market, leveraging Langchain.
-- Provides a ready-to-deploy frontend app to showcase the E2E user workflow.
+- Cross platform digital application powered by [FlutterFlow](https://flutterflow.io/) that delivers seamless experience to users on Android, iOS, web, and desktop platforms.
```
Comment (jonchenn) [inline README.md:28]:
```suggestion
- Cross platform frontend application powered by [FlutterFlow](https://flutterflow.io/) that delivers end-to-end user flows and seamless experience on Android, iOS, web, and desktop platforms.
```

**Review by jonchenn — CHANGES_REQUESTED** [review]
I'd suggest pointing out the "frontend app" here to make it clear this is the FE/UI app used in this repo.

File: README.md (hunk 1 above)
Comment (kawofong) [reply README.md]: Updated!

**Review by jonchenn — APPROVED** [review]
LGTM, thanks for working on this!

**Pull request merged**